src/BloomExe/NavigationIsolator.cs

```diff
@@ -148,11 +148,21 @@ private void BrowserOnDocumentCompleted(object sender, EventArgs eventArgs)
 		private void ForceDocumentCompleted()
 		{
 			Cleanup();
-			if (_pending.Count > 0)
+			while (_pending.Count > 0)
 			{
 				var task = _pending[0];
 				_pending.RemoveAt(0);
+				// Somehow raw files in /tmp can disappear after being sent to NavigationIsolator.Navigate() and before
+				// being displayed via the pending mechanism.  If the file doesn't exist, calling _current.Browser.Navigate()
+				// results in the program silently stopping on Linux.  (Going "poof" is how I would describe it.)
+				// See https://jira.sil.org/browse/BL-863 for a bug report.
+				if (task.Url.StartsWith("/tmp/") && !System.IO.File.Exists(task.Url))
+				{
+					Debug.WriteLine(String.Format("DEBUG: NavigationIsolator.ForceDocumentCompleted(): new file to display (\"{0}\") does not exist!??", task.Url));
```

Comment:
A Debug.Assert would be rather annoying and overly scary on Linux developer machines, and wouldn't provide any information on user machines (which generally have Release builds).  I could make it a Console.WriteLine() instead so that the message appears on user machines and developer machines, but didn't want to scare users/testers who happened to start the program from the command line.  The latter reason may not be too significant since it does start off saying DEBUG.  :-)